node: Governor token list update

djb15 opened the pull request:
```
Tokens before = 1599
Tokens after = 1623

Tokens added = 24:
<WH_chain_id>-<WH_token_addr>-<token_symbol>

[
 "1-00587a642600d6f2bd3bbfdf116dbd613c45cfa659b55a1fbc7cbc28e6092699-Skibidi",
 "1-07bb093e9f7decab41a717b15946f6db587868a6721c0a6e3c4281ed3fef0e09-XBG",
 "1-0e21542327931809efbf3b282c09eb7ef5f82c68e5c35024c97da4f89a2234c7-DONK",
 "1-20dc94caa1c4d02e8d2d5d955438684fa2fc1558a68ffe079b4b813554301119-HM",
 "1-6270a9646bd0d4a8daeb0deba12c396b88fa79f4bf3c16f844ed91af004a4723-GCR",
 "1-6b04a01bfd79e1eddf99d77598a5da54ee3cc8189dedfc1408908e29e1e94058-POWSCHE",
 "1-826d35ab8f5b30c7d511f8f2ba9d8765228e94c8b810afc82c9a3a29a8146f89-FLOCKA",
 "1-8300ec316d7d1c7a81a6907c633655bcf37368889c2bed5f47901d0f56e01c8b-KEYCAT",
 "1-b41652c46fff703ca2d2d9660dc6e9c2f9eb59cbd77ba07d1edf3548c30e4fac-CAIR",
 "1-b5cbeacdc2c4b9172fc2055facc3b365eacb144ec670fcd02689fc9c29a252be-KWEEN",
 "1-b814cbaa6004e889059e49841809b54c120f6d17c37cfaace4d28d394d56f67c-SKBDI",
 "1-c206c5c565936e75a63aca8d62d7f2c27c21572474d7dd13e38424b0377f17af-ADA",
 "1-ceaf14fcf7cb2ed5003289247656f3427aa86c1cee4185effcc253e7a8fe403a-LAIKA",
 "2-0000000000000000000000008457ca5040ad67fdebbcc8edce889a335bc0fbfb-ALT",
 "2-0000000000000000000000009dfad1b7102d46b1b197b90095b5c4e9f5845bba-BOTTO",
 "2-0000000000000000000000009f52c8ecbee10e00d9faaac5ee9ba0ff6550f511-SIPHER",
 "4-000000000000000000000000031b41e504677879370e9dbcf937283a8691fa7f-FET",
 "5-0000000000000000000000005f0197ba06860dac7e31258bdf749f92b6a636d4-1FLR",
 "5-000000000000000000000000bd1463f02f61676d53fd183c2b19282bff93d099-jCHF",
 "5-000000000000000000000000e4bf2864ebec7b7fdf6eeca9bacae7cdfdaffe78-DODO",
 "22-69e4491bcf06bae41346240afbdab99cdbe40b38d2bb22eb1cf9b4a4d9e635f8-MBX",
 "23-00000000000000000000000088266f9eb705f5282a2507a9c418821a2ac9f8bd-NCASH",
 "30-000000000000000000000000afb89a09d82fbde58f18ac6437b3fc81724e4df6-DOG",
 "30-000000000000000000000000f7c1cefcf7e1dd8161e00099facd3e1db9e528ee-TOWER"
]

Tokens removed = 0:
<WH_chain_id>-<WH_token_addr>-<token_symbol>

[]

Tokens with changed symbols = 1:
<WH_chain_id>-<WH_token_addr>-<old_token_symbol>-><new_token_symbol>

[
 "5-0000000000000000000000000d500b1d8e8ef31e21c99d1db9a6444d3adf1270-WMATIC->WPOL"
]

Tokens with invalid symbols = 0:
<WH_chain_id>-<WH_token_addr>-<token_symbol>

[]

Potentially depegged USD stablecoins (>10%) = 13:
<WH_chain_id>-<WH_token_addr>-<token_symbol> = <token_price>

[
 "1-689ac099ef657e5d3b7efaf1e36ab8b897e2746232d8a9261b3e49b35c1dead4-xUSD = 0.59214",
 "2-000000000000000000000000309627af60f0926daa6041b8279484312f2bf060-USDB = 4.827e-9",
 "2-000000000000000000000000df574c24545e5ffecb9a659c229253d4111d87e1-HUSD = 0.02807749",
 "4-00000000000000000000000023e8a70534308a4aaf76fb8c32ec13d17a3bd89e-lUSD = 0.00003213",
 "4-000000000000000000000000de7d1ce109236b12809c45b23d22f30dba0ef424-USDS = 0.112356",
 "10-000000000000000000000000cc1b99ddac1a33c201a742a1851662e87bc7f22c-USDT = 0.733349",
 "11-0000000000000000000000000000000000000000000100000000000000000081-aUSD = 0.51763",
 "12-0000000000000000000000000000000000000000000100000000000000000001-aUSD = 0.450452",
 "13-0000000000000000000000005c74070fdea071359b86082bd9f9b3deaafbe32b-KDAI = 0.369392",
 "13-000000000000000000000000754288077d0ff82af7a5317c7cb8c444d421d103-oUSDC = 0.294212",
 "13-000000000000000000000000cee8faf64bb97a73bb51e115aa89c17ffa8dd167-oUSDT = 0.133103",
 "16-000000000000000000000000818ec0a7fe18ff94269904fced6ae3dae6d6dc0b-USDC = 0.134078",
 "16-000000000000000000000000ffffffff52c56a9257bb97f4b2b6f7b2d624ecda-xcaUSD = 0.450452"
]

Tokens with significant price drops (>50%) = 83:

[
 {
  "token": "1-052e7f80a14a1e7a5254e9a1c27f7fc48563bfc35f0be177a8d556ae8ba1add6-MEAN",
  "previousPrice": 0.00731877,
  "newPrice": 0.00088285,
  "percentageChange": "-87.9"
 },
 {
  "token": "1-067fc27abcad2df07cc40437330da4fe8851680ae2b242c2ea1d86e2cfa10064-SNS",
  "previousPrice": 0.03561527,
  "newPrice": 0.01384187,
  "percentageChange": "-61.1"
 },
 {
  "token": "1-0b8bba91467afe97500d18b35605e1c5842b2310a97ebfb863d17f32f1943fb1-PEEP",
  "previousPrice": 0.0000115,
  "newPrice": 0.00000454,
  "percentageChange": "-60.5"
 },
 {
  "token": "1-2ccec8f92e8942182bef15efc347d609a7b2010db68d678664fae043b5f6e707-BOGUS",
  "previousPrice": 0.00242705,
  "newPrice": 0.00064923,
  "percentageChange": "-73.3"
 },
 {
  "token": "1-2f64669f9f8ad5ee900c4050cf06c756f3d10a02c908f803c1e43ed7a145d3a0-SWOLE",
  "previousPrice": 0.0000068,
  "newPrice": 0.0000032,
  "percentageChange": "-52.9"
 },
 {
  "token": "1-4262f09434e938b2714803d3aa425d1d543a735ba324109087760d83deae653d-SPWN",
  "previousPrice": 0.00004114,
  "newPrice": 0.00001587,
  "percentageChange": "-61.4"
 },
 {
  "token": "1-5468187df701e428898565f1c522b352ad481d0dcb40d8b6e1036ba153e5e3af-pog",
  "previousPrice": 0.00121212,
  "newPrice": 0.00036143,
  "percentageChange": "-70.2"
 },
 {
  "token": "1-69302bef50efd86a9bc1c2ac763ff56633e9fda1709eb0b858088e9cfbdcfe0f-Rizzmas",
  "previousPrice": 0.00003163,
  "newPrice": 0.0000032,
  "percentageChange": "-89.9"
 },
 {
  "token": "1-6b054395432c64d8250648371121f271fc3b8e36621384b9e91956bf02307c13-DOKI",
  "previousPrice": 0.00042,
  "newPrice": 0.00020869,
  "percentageChange": "-50.3"
 },
 {
  "token": "1-6dbb85629db6a38eeb2fec793974b4c7e9c5fdc9e70165303b4f73106e5e62ba-",
  "previousPrice": 0.02526629,
  "newPrice": 0.01116572,
  "percentageChange": "-55.8"
 },
 {
  "token": "1-842f35b8d1f37e6cc0a7eb8d12bf9079f7abf194cdcd466c22f5f1e8baa4dea8-",
  "previousPrice": 0.0002046,
  "newPrice": 0.00009927,
  "percentageChange": "-51.5"
 },
 {
  "token": "1-86d536568df6b574b219c07565198d7fab61cb0458cdfbadf0f004759f2e5038-GMEOW",
  "previousPrice": 0.00016373,
  "newPrice": 0.00005906,
  "percentageChange": "-63.9"
 },
 {
  "token": "1-9353906bba080336a76db51391c3737194d3a31183665f4b7021e908739bc0f2-COST",
  "previousPrice": 0.00388238,
  "newPrice": 0.00137439,
  "percentageChange": "-64.6"
 },
 {
  "token": "1-969c68e974ffbb74feed1af1672b9f67adb65fb382ff2ee105070fd8d60d9fe5-POS",
  "previousPrice": 0.00009495,
  "newPrice": 0.00004403,
  "percentageChange": "-53.6"
 },
 {
  "token": "1-a084edb439cccf78ff5db4936074779133cc49d8058731fb9ad6d982f57f899e-BOBO",
  "previousPrice": 1.22807e-7,
  "newPrice": 5.9525e-8,
  "percentageChange": "-51.5"
 },
 {
  "token": "1-ced51ea8ae7d7670ce24e31fa7cc1f731ee3e02b91c017a37af71968cb17a290-CHADCAT",
  "previousPrice": 0.00022358,
  "newPrice": 0.00008347,
  "percentageChange": "-62.7"
 },
 {
  "token": "1-d6c679aa7652867ebb3221cd90f5cf1d7f1b20e3bfd713c95de54bbdd6df2172-wBIRD",
  "previousPrice": 2.04,
  "newPrice": 0.856385,
  "percentageChange": "-58.0"
 },
 {
  "token": "2-00000000000000000000000000e679ba63b509182c349f5614f0a07cdd0ce0c5-DAMEX",
  "previousPrice": 0.00195043,
  "newPrice": 0.00089854,
  "percentageChange": "-53.9"
 },
 {
  "token": "2-0000000000000000000000000b61c4f33bcdef83359ab97673cb5961c6435f4e-EARN",
  "previousPrice": 0.00469054,
  "newPrice": 0.00177104,
  "percentageChange": "-62.2"
 },
 {
  "token": "2-00000000000000000000000024249b5a869a445c9b0ce269a08d73c618df9d21-ETHEREUM",
  "previousPrice": 0.00118558,
  "newPrice": 0.00049162,
  "percentageChange": "-58.5"
 },
 {
  "token": "2-0000000000000000000000002c8ea636345a231e4b1a28f6eeb2072ed909c406-MemElon",
  "previousPrice": 5.2038e-11,
  "newPrice": 2.1905e-11,
  "percentageChange": "-57.9"
 },
 {
  "token": "2-000000000000000000000000389999216860ab8e0175387a0c90e5c52522c945-FEG",
  "previousPrice": 2.87583e-10,
  "newPrice": 8.8569e-11,
  "percentageChange": "-69.2"
 },
 {
  "token": "2-0000000000000000000000004674672bcddda2ea5300f5207e1158185c944bc0-GXT",
  "previousPrice": 0.00010226,
  "newPrice": 0.00001828,
  "percentageChange": "-82.1"
 },
 {
  "token": "2-000000000000000000000000490e3f4af13e1616ec97a8c6600c1061a8d0253e-TRR",
  "previousPrice": 0.0010422,
  "newPrice": 0.00029301,
  "percentageChange": "-71.9"
 },
 {
  "token": "2-00000000000000000000000070401dfd142a16dc7031c56e862fc88cb9537ce0-BIRD",
  "previousPrice": 2.04,
  "newPrice": 0.856385,
  "percentageChange": "-58.0"
 },
 {
  "token": "2-0000000000000000000000007697b462a7c4ff5f8b55bdbc2f4076c2af9cf51a-SARCO",
  "previousPrice": 0.00649969,
  "newPrice": 0.00083668,
  "percentageChange": "-87.1"
 },
 {
  "token": "2-0000000000000000000000008287c7b963b405b7b8d467db9d79eec40625b13a-SWINGBY",
  "previousPrice": 0.00050987,
  "newPrice": 0.00016313,
  "percentageChange": "-68.0"
 },
 {
  "token": "2-0000000000000000000000008a854288a5976036a725879164ca3e91d30c6a1b-GET",
  "previousPrice": 1.48,
  "newPrice": 0.736479,
  "percentageChange": "-50.2"
 },
 {
  "token": "2-0000000000000000000000008ab7404063ec4dbcfd4598215992dc3f8ec853d7-AKRO",
  "previousPrice": 0.00459367,
  "newPrice": 0.00088488,
  "percentageChange": "-80.7"
 },
 {
  "token": "2-0000000000000000000000008c223a82e07fecb49d602150d7c2b3a4c9630310-NFTE",
  "previousPrice": 0.00005538,
  "newPrice": 0.00001848,
  "percentageChange": "-66.6"
 },
 {
  "token": "2-000000000000000000000000aa2ce7ae64066175e0b90497ce7d9c190c315db4-Suter",
  "previousPrice": 0.00004452,
  "newPrice": 0.00002117,
  "percentageChange": "-52.4"
 },
 {
  "token": "2-000000000000000000000000b131f337c45d386ceec234e194b2663d5c3d9dcf-ICOM",
  "previousPrice": 0.01032648,
  "newPrice": 0.0031347,
  "percentageChange": "-69.6"
 },
 {
  "token": "2-000000000000000000000000bededdf2ef49e87037c4fb2ca34d1ff3d3992a11-FEG",
  "previousPrice": 0.00036373,
  "newPrice": 0.00014948,
  "percentageChange": "-58.9"
 },
 {
  "token": "2-000000000000000000000000c0a4df35568f116c370e6a6a6022ceb908eeddac-UMEE",
  "previousPrice": 0.00147932,
  "newPrice": 0.00067391,
  "percentageChange": "-54.4"
 },
 {
  "token": "2-000000000000000000000000c28eb2250d1ae32c7e74cfb6d6b86afc9beb6509-OPN",
  "previousPrice": 0.0014495,
  "newPrice": 0.00071188,
  "percentageChange": "-50.9"
 },
 {
  "token": "2-000000000000000000000000c97d6c52f3add91fa1c5287a453d7444aecbca83-DZOO",
  "previousPrice": 0.00444105,
  "newPrice": 0.002038,
  "percentageChange": "-54.1"
 },
 {
  "token": "2-000000000000000000000000ce246eea10988c495b4a90a905ee9237a0f91543-VCX",
  "previousPrice": 0.03852542,
  "newPrice": 0.01686005,
  "percentageChange": "-56.2"
 },
 {
  "token": "2-000000000000000000000000dd69db25f6d620a7bad3023c5d32761d353d3de9-GETH",
  "previousPrice": 0.02048946,
  "newPrice": 0.00671868,
  "percentageChange": "-67.2"
 },
 {
  "token": "2-000000000000000000000000dfdb7f72c1f195c5951a234e8db9806eb0635346-NFD",
  "previousPrice": 0.00030757,
  "newPrice": 0.00014718,
  "percentageChange": "-52.1"
 },
 {
  "token": "2-000000000000000000000000e020b01b6fbd83066aa2e8ee0ccd1eb8d9cc70bf-ARCD",
  "previousPrice": 0.02887222,
  "newPrice": 0.01346495,
  "percentageChange": "-53.4"
 },
 {
  "token": "2-000000000000000000000000e1c7e30c42c24582888c758984f6e382096786bd-XCUR",
  "previousPrice": 0.00494438,
  "newPrice": 0.00245587,
  "percentageChange": "-50.3"
 },
 {
  "token": "2-000000000000000000000000e516d78d784c77d479977be58905b3f2b1111126-SPWN",
  "previousPrice": 0.00004114,
  "newPrice": 0.00001587,
  "percentageChange": "-61.4"
 },
 {
  "token": "2-000000000000000000000000eb4c2781e4eba804ce9a9803c67d0893436bb27d-renBTC",
  "previousPrice": 91170,
  "newPrice": 33929,
  "percentageChange": "-62.8"
 },
 {
  "token": "2-000000000000000000000000f2fdd9c25d7bc8002ce89716d1be484b2d976944-𝕏PAY",
  "previousPrice": 0.00269228,
  "newPrice": 0.00037109,
  "percentageChange": "-86.2"
 },
 {
  "token": "3-00000000000000000000000034aa51100370594b15479392149157eeb479b9a8-ASTRO",
  "previousPrice": 0.00144144,
  "newPrice": 0.00020221,
  "percentageChange": "-86.0"
 },
 {
  "token": "4-0000000000000000000000000a356f512f6fce740111ee04ab1699017a908680-UFARM",
  "previousPrice": 0.00016965,
  "newPrice": 0.00007284,
  "percentageChange": "-57.1"
 },
 {
  "token": "4-0000000000000000000000001591e923e0836a3949b59637fbe8959f000894b9-MAI",
  "previousPrice": 0.00203575,
  "newPrice": 0.00065357,
  "percentageChange": "-67.9"
 },
 {
  "token": "4-00000000000000000000000021f9b5b2626603e3f40bfc13d01afb8c431d382f-ZINU",
  "previousPrice": 2.80144e-10,
  "newPrice": 4.5949e-11,
  "percentageChange": "-83.6"
 },
 {
  "token": "4-0000000000000000000000002326c7395d02a8c89a9d7a0b0c1cf159d49ce51c-TBAC",
  "previousPrice": 0.00976581,
  "newPrice": 0.00414222,
  "percentageChange": "-57.6"
 },
 {
  "token": "4-000000000000000000000000245d9f531757f83064ad808b4c9b220c703a4934-GODE",
  "previousPrice": 0.00084764,
  "newPrice": 0.00038819,
  "percentageChange": "-54.2"
 },
 {
  "token": "4-0000000000000000000000006d57f5c286e04850c2c085350f2e60aaa7b7c15b-GrokGirl",
  "previousPrice": 2.8301e-13,
  "newPrice": 7.1635e-14,
  "percentageChange": "-74.7"
 },
 {
  "token": "4-0000000000000000000000008ebc361536094fd5b4ffb8521e31900614c9f55d-DARC",
  "previousPrice": 0.00234057,
  "newPrice": 0.00069774,
  "percentageChange": "-70.2"
 },
 {
  "token": "4-000000000000000000000000935a544bf5816e3a7c13db2efe3009ffda0acda2-BLZ",
  "previousPrice": 0.145078,
  "newPrice": 0.053927,
  "percentageChange": "-62.8"
 },
 {
  "token": "4-000000000000000000000000959229d94c9060552daea25ac17193bca65d7884-IOI",
  "previousPrice": 0.01931646,
  "newPrice": 0.00854447,
  "percentageChange": "-55.8"
 },
 {
  "token": "4-0000000000000000000000009e6b3e35c8f563b45d864f9ff697a144ad28a371-DOGO",
  "previousPrice": 5.63585e-7,
  "newPrice": 1.96155e-7,
  "percentageChange": "-65.2"
 },
 {
  "token": "4-000000000000000000000000a58950f05fea2277d2608748412bf9f802ea4901-WSG",
  "previousPrice": 7.28116e-10,
  "newPrice": 2.30122e-10,
  "percentageChange": "-68.4"
 },
 {
  "token": "4-000000000000000000000000c342774492b54ce5f8ac662113ed702fc1b34972-BGEO",
  "previousPrice": 0.00156339,
  "newPrice": 0.00019776,
  "percentageChange": "-87.4"
 },
 {
  "token": "4-000000000000000000000000fce146bf3146100cfe5db4129cf6c82b0ef4ad8c-renBTC",
  "previousPrice": 91170,
  "newPrice": 33929,
  "percentageChange": "-62.8"
 },
 {
  "token": "4-000000000000000000000000fdfd27ae39cebefdbaac8615f18aa68ddd0f15f5-GHD",
  "previousPrice": 0.00000229,
  "newPrice": 4.48473e-7,
  "percentageChange": "-80.4"
 },
 {
  "token": "5-000000000000000000000000011734f6ed20e8d011d85cf7894814b897420acf-ACRE",
  "previousPrice": 0.00016442,
  "newPrice": 0.00003292,
  "percentageChange": "-80.0"
 },
 {
  "token": "5-0000000000000000000000002ad2934d5bfb7912304754479dd1f096d5c807da-AGC",
  "previousPrice": 0.339441,
  "newPrice": 0.141377,
  "percentageChange": "-58.4"
 },
 {
  "token": "5-0000000000000000000000007844f79fc841e4f92d974c417031c76f8578c2d5-OPN",
  "previousPrice": 0.0014495,
  "newPrice": 0.00071188,
  "percentageChange": "-50.9"
 },
 {
  "token": "5-000000000000000000000000aa404804ba583c025fa64c9a276a6127ceb355c6-CPR",
  "previousPrice": 0.00121227,
  "newPrice": 0.00029895,
  "percentageChange": "-75.3"
 },
 {
  "token": "5-000000000000000000000000bbfe0b60de96a189bf09079de86a2db7bf0c7883-eLunr",
  "previousPrice": 0.02584591,
  "newPrice": 0.00862433,
  "percentageChange": "-66.6"
 },
 {
  "token": "5-000000000000000000000000efee2de82343be622dcb4e545f75a3b9f50c272d-TRY",
  "previousPrice": 0.00025119,
  "newPrice": 0.00000877,
  "percentageChange": "-96.5"
 },
 {
  "token": "8-000000000000000000000000000000000000000000000000000000000d83314a-YLDY",
  "previousPrice": 0.00004567,
  "newPrice": 0.00002083,
  "percentageChange": "-54.4"
 },
 {
  "token": "10-00000000000000000000000085dec8c4b2680793661bca91a8f129607571863d-BRUSH",
  "previousPrice": 0.03549674,
  "newPrice": 0.01670095,
  "percentageChange": "-53.0"
 },
 {
  "token": "10-00000000000000000000000097bdafe3830734acf12da25359674277fcc33729-KIRBY",
  "previousPrice": 0.00008521,
  "newPrice": 0.0000204,
  "percentageChange": "-76.1"
 },
 {
  "token": "10-000000000000000000000000b715f8dce2f0e9b894c753711bd55ee3c04dca4e-CONK",
  "previousPrice": 2.03989e-10,
  "newPrice": 9.0155e-11,
  "percentageChange": "-55.8"
 },
 {
  "token": "10-000000000000000000000000be41772587872a92184873d55b09c6bb6f59f895-MARS",
  "previousPrice": 4.21487e-7,
  "newPrice": 1.42964e-7,
  "percentageChange": "-66.1"
 },
 {
  "token": "10-000000000000000000000000e47d957f83f8887063150aaf7187411351643392-CHILL",
  "previousPrice": 5.72156e-10,
  "newPrice": 1.9588e-11,
  "percentageChange": "-96.6"
 },
 {
  "token": "10-000000000000000000000000e64b9fd040d1f9d4715c645e0d567ef69958d3d9-MOD",
  "previousPrice": 0.03611367,
  "newPrice": 0.01292186,
  "percentageChange": "-64.2"
 },
 {
  "token": "14-00000000000000000000000000be915b9dcf56a3cbe739d9b9c202ca692409ec-UBE",
  "previousPrice": 0.00025388,
  "newPrice": 0.00012277,
  "percentageChange": "-51.6"
 },
 {
  "token": "14-00000000000000000000000046c9757c5497c5b1f2eb73ae79b6b67d119b0b58-PACT",
  "previousPrice": 0.00002924,
  "newPrice": 6.0239e-7,
  "percentageChange": "-97.9"
 },
 {
  "token": "15-152fd354c086478f67c36eaa07d209631825c097490fbdb38b2a9876039be6e0-PURGE",
  "previousPrice": 0.0242359,
  "newPrice": 0.01067331,
  "percentageChange": "-56.0"
 },
 {
  "token": "23-00000000000000000000000051b902f19a56f0c8e409a34a215ad2673edf3284-NFTE",
  "previousPrice": 0.00005538,
  "newPrice": 0.00001848,
  "percentageChange": "-66.6"
 },
 {
  "token": "23-000000000000000000000000a684cd057951541187f288294a1e1c2646aa2d24-VSTA",
  "previousPrice": 0.169467,
  "newPrice": 0.051561,
  "percentageChange": "-69.6"
 },
 {
  "token": "23-000000000000000000000000b261104a83887ae92392fb5ce5899fcfe5481456-NFTE",
  "previousPrice": 0.00005538,
  "newPrice": 0.00001848,
  "percentageChange": "-66.6"
 },
 {
  "token": "23-000000000000000000000000f8388c2b6edf00e2e27eef5200b1befb24ce141d-NOLA",
  "previousPrice": 0.00012228,
  "newPrice": 0.00000311,
  "percentageChange": "-97.5"
 },
 {
  "token": "30-00000000000000000000000078a087d713be963bf307b18f2ff8122ef9a63ae9-BSWAP",
  "previousPrice": 0.737613,
  "newPrice": 0.341692,
  "percentageChange": "-53.7"
 },
 {
  "token": "30-0000000000000000000000007f12d13b34f5f4f0a9449c16bcd42f0da47af200-NORMIE",
  "previousPrice": 0.00974977,
  "newPrice": 0.00306409,
  "percentageChange": "-68.6"
 },
 {
  "token": "30-0000000000000000000000008e16d46cb2da01cdd49601ec73d7b0344969ae33-coin",
  "previousPrice": 0.00000525,
  "newPrice": 0.00000257,
  "percentageChange": "-51.0"
 },
 {
  "token": "30-000000000000000000000000d07379a755a8f11b57610154861d694b2a0f615a-BASE",
  "previousPrice": 0.00001944,
  "newPrice": 0.00000888,
  "percentageChange": "-54.3"
 }
]
```

djb15 commented:
The `USDB` price here is very wrong and also isn't reflected in the CoinGecko graph, need to take a look

johnsaigle commented:
@djb15 

```
 rg 'usd-bancor' ../../pkg/governor/generated_mainnet_tokens.go
364:            {chain: 2, addr: "000000000000000000000000309627af60f0926daa6041b8279484312f2bf060", symbol: "USDB", coinGeckoId: "usd-bancor", decimals: 18, price: 4.827e-9},                                         // Addr: 0x309627af60f0926daa6041b8279484312f2bf060, Notional: 0.00000246177
```

If I look up `usd-bancor` (https://www.coingecko.com/en/coins/usd-bancor) CoinGecko says that it has been deactivated. Note that this shouldn't be confused with USDB on Blast.

Can we manually remove this token in this case?

Also I'm now wondering we should check for deactivated tokens somehow. (In `check_prices.go` or else in the 'hack' TS script)

djb15 commented:
Ah thanks @johnsaigle, hadn't had a chance to look at this yet but yeah it wasn't Blast like I thought. Pretty tedious but I went through the list manually and updated the ignore list in the script.

The good news is this check actually caught a legit depeg of `10-000000000000000000000000cc1b99ddac1a33c201a742a1851662e87bc7f22c-USDT`!

I've also updated the codeowners for the `node/hack/governor` folder to save folks from getting pulled in to these types of changes in the future; it matches the codeowners for `node/pkg/governor`.

bruce-riley reviewed (approved):
Only reviewed the code owners and script, not the generated tokens.

evan-gray reviewed (approved):
approving owners change
